packages/react-recommendations/src/utils/sortBy.ts

```diff
@@ -0,0 +1,8 @@
+type Predicate<TITem> = (a: TITem, b: TITem) => number;
```

Comment:
We should probably stick to `TObject` since it's already implemented in the codebase, wdyt?

Reply:
These utils are supposed to be used with any data structure so there's no strict reason to use the same naming as the generic Algolia records. But it should be cased `TItem` indeed.

Reply:
Make sense!

Reply:
See f54ef351295225772dfb819f15bfdc1691af6b14.